evcache-core/src/test/java/com/netflix/evcache/test/Base.java

```diff
@@ -239,6 +248,13 @@ public Map<String, String> getBulk(String keys[], EVCache gCache) throws Excepti
         return value;
     }
 
+    public Map<String, String> getCompletableBulk(String keys[], EVCache gCache) throws Exception {
+        ExecutorService executorService = Executors.newFixedThreadPool(10);
+        final CompletableFuture<Map<String, String>> value = gCache.<String>getBulk(executorService, keys);
+        if(log.isDebugEnabled()) log.debug("getBulk : keys : " + Arrays.toString(keys) + "; values = " + value);
+        return value.get();
```

Comment:
Can we add a test case that would use this in an async manner? You have to wait at a different level to ensure results are returned as expected, but not using the `get` method.